### PR TITLE
This will add BIGIP-13.0.0 image to driver systests

### LINKED ISSUE #628: BIGIP-13.0.0 image onboarding for driver tests
* Title: BIGIP-13.0.0 image onboarding for driver tests
* Details: Edit the systest Makefile to incorporate BIGIP-13.0.0 image's testing

#### Agent Version
8.x.x, 9.x.x

#### Operating System
CentOS for testing

#### OpenStack Release
Liberty && Mitaka

#### OpenStack Release
Liberty

#### Description
This is to incorporate the newly onboarded BIGIP-13.0.0 image build 1645.  This image will be used for testing purposes on the driver.

### PR DESCRIPTION
**Pull request by ssorenso**
Issues:
Fixes #628

Problem:
* BIGIP-13.0.0 is currently not covered in driver tests

Analysis:
* This change will allow nightly to include the driver tests for
  * overcloud standard
  * undercloud_vlan
  * undercloud_gre
  * undercloud standard

Tests:
This is a Makefile that wil initiate the tests for BIGIP-13.

@zancas 
#### What issues does this address?
Fixes #628 

#### What's this change do?
* This change will allow nightly to include the driver tests for
  * overcloud standard
  * undercloud_vlan
  * undercloud_gre
  * undercloud standard

#### Where should the reviewer start?
* At the commits starting with the Makefile

#### Any background context?
By and large, not really besides the knowledge of nightly systests.

### CODE REVIEW

File: systest/Makefile [hunk 1]
```diff
@@ -17,8 +17,11 @@
 .PHONY: all functest \
 install_tlc \
 install_test_infra \
+install_jenkins \
```
Comment (zancas) [inline systest/Makefile]: This isn't how we manage jenkins in production.

File: systest/Makefile [hunk 2]
```diff
@@ -104,6 +107,11 @@ export FROM_AGENT_SESSION := from.agent_$(SUBJECTCODE_ID)_$(TIMESTAMP)
 # without exception, invoke the aforementioned scripts.
 #
 
+install_jenkins:
+	useradd -ms /bin/bash jenkins
+	cp -r /home/buildbot/.ssh/ /home/jenkins/
+	chown -R jenkins:jenkins /home/jenkins/.ssh/
+
```
Comment (zancas) [inline systest/Makefile]: Please remove.

File: systest/Makefile [hunk 3]
```diff
@@ -138,9 +146,37 @@ cleanup_failed_tlc:
...
+functest: install_tlc install_test_infra
+	-$(MAKE) -C . tempest_12.1.1_overcloud
+	-$(MAKE) -C . tempest_13.0.0_overcloud
+	-$(MAKE) -C . tempest_12.1.1_undercloud
+	-$(MAKE) -C . tempest_12.1.1_undercloud_gre
+	-$(MAKE) -C . tempest_12.1.1_undercloud_vlan
+	-$(MAKE) -C . tempest_13.0.0_undercloud
+	-$(MAKE) -C . tempest_13.0.0_undercloud_gre
+	-$(MAKE) -C . tempest_13.0.0_undercloud_vlan
+
```
Comment (zancas) [inline systest/Makefile]: Please don't add back this old pattern.

Comment (ssorenso) [inline systest/Makefile]: what pattern should I use, then? I *thought* I was simply to repeat patterns that exist...

**Pull request merged**